GeneralsMD/Code/Tools/wdump/chunk_d.cpp

```diff
@@ -2057,6 +2116,10 @@ ChunkTableClass::ChunkTableClass() {
 	NewType( W3D_CHUNK_VERTEX_SHADE_INDICES, "W3D_CHUNK_VERTEX_SHADE_INDICES",List_W3D_CHUNK_VERTEX_SHADE_INDICES);
 	NewType( W3D_CHUNK_MATERIAL_INFO,"W3D_CHUNK_MATERIAL_INFO",List_W3D_CHUNK_MATERIAL_INFO);
 	NewType( W3D_CHUNK_SHADERS,"W3D_CHUNK_SHADERS",List_W3D_CHUNK_SHADERS);
+	NewType( W3D_CHUNK_FX_SHADERS,"W3D_CHUNK_FX_SHADERS",List_W3D_CHUNK_FX_SHADERS, true);
+	NewType( W3D_CHUNK_FX_SHADER,"W3D_CHUNK_FX_SHADER",List_W3D_CHUNK_FX_SHADER, true);
+	NewType( W3D_CHUNK_FX_SHADER_INFO, "W3D_CHUNK_FX_SHADER_INFO", List_W3D_CHUNK_FX_SHADER_INFO);
+	NewType( W3D_CHUNK_FX_SHADER_CONSTANT, "W3D_CHUNK_FX_SHADER_CONSTANT", List_W3D_CHUNK_FX_SHADER_CONSTANT);
```

Comment:
wdump understands these chunks now. Does the game understand them as well, or not? If not, are there future plans for it or is the plan that only the tool can understand it?